controllers/request.js

```diff
@@ -0,0 +1,15 @@
+import models from '../models';
+
+const { Request } = models;
+
+export default {
+  async requestMentorship(req, res) {
```

Comment:
This might sound like asking for a mentor. Could you change to `requestToBeAMentor`? or something related?

Reply:
I have done that